packages/legend-manual-tests/src/__tests__/roundtrip-grammar/cases/DSL_Persistence-basic.pure

```diff
@@ -172,6 +172,10 @@ Persistence org::legend::ZooPersistence
       }
     ];
   }
+  tests:
```

Comment:
You may want to fix this in `engine` composer
You don't want to render the tests part of the grammar if there is no tests.

Reply:
Isn't TestSuite the same? tried to have some consistency over there.